server/services/communityApiService.ts

```diff
@@ -22,6 +22,13 @@ export default class CommunityApiService {
     return (await this.restClient.get({ path: `/secure/offenders/crn/${crn}`, token })) as DeliusServiceUser
   }
 
+  async getExpandedServiceUserByCRN(crn: string): Promise<ExpandedDeliusServiceUser> {
+    const token = await this.hmppsAuthService.getApiClientToken()
+
+    logger.info({ crn }, 'getting all details for offender')
+    return (await this.restClient.get({ path: `/secure/offenders/crn/${crn}/all`, token })) as ExpandedDeliusServiceUser
```

Comment:
could we wrap rest client errors with a user message that says something about 'could not get service user details from nDelius'?

it would be

try {
  ... 
} catch (err) {
  throw createError(err.status, err, { userMessage: '...' })
}

Reply:
Yep, sure.